### DISCUSSION TITLE
Update DNF dependencies (Wiki)

### DISCUSSION THREAD

**Discussion by hus3h** [polybar/polybar#3000] [Build Support]
Tested on Fedora 38 in a [toolbox](https://docs.fedoraproject.org/en-US/fedora-silverblue/toolbox/) container. I think more optional dependencies are needed for the pulseaudio (pipewire-pulseaudio?) and network internal modules to work when manually compiling, but didn't figure them out yet. These changes might be relevant for openSUSE too, but I don't have a setup to test them there.

Note: `wireless-tools-devel` doesn't seem to be available anymore, last I could find is for Fedora 36 (current Fedora version is 38).

[polybar-dnf-changes-patch.txt](https://github.com/polybar/polybar/files/12213212/polybar-dnf-changes-patch.txt)

**Comment by patrick96** [comment]
Appreciate the effort! I added your changes to the wiki.

> Note: wireless-tools-devel doesn't seem to be available anymore, last I could find is for Fedora 36 (current Fedora version is 38).

wireless-tools have been deprecated for a while now, so no surprise that they're no longer in newer fedora versions.

> I think more optional dependencies are needed for the pulseaudio (pipewire-pulseaudio?) and network internal modules to work when manually compiling, but didn't figure them out yet.

According to the spec file for the [official polybar pakcage](https://src.fedoraproject.org/rpms/polybar/blob/rawhide/f/polybar.spec). It simply requires `libnl3-devel` for the network module. For pulseaudio, it requires on the `libpulse` pkgconfig, looking at the build logs and package content, the `pulseaudio-libs-devel` should also provide this.

**Comment by hus3h** [comment]
Thanks! Just tried the updated commands and everything seems fine including the network and pulseaudio modules.
